package.json

```diff
@@ -50,7 +50,8 @@
     "electron-compile": "^6.4.4",
     "electron-squirrel-startup": "^1.0.0",
     "nedb": "^1.8.0",
-    "nedb-promise": "^2.0.1"
+    "nedb-promise": "^2.0.1",
+    "pug": "^2.0.4"
```

Comment:
This should be a dev dependency since it is not needed after the html file is compiled